### ISSUE TITLE
attribute `lib` missing

### ISSUE THREAD

**Issue by savoisn** [Elyhaka/shadow-nix#9]
Trying to use it in home-manager I get this error

I am using home-manager
and it's giving me this error.

```
error: attribute 'lib' missing, at /nix/store/71m527m700yccimfi3pqfaknzwzblf5g-source/shadow-package.nix:70:5
(use '--show-trace' to show detailed location information)
```
I am probably doing something wrong with my home-manager conf... 
Any idea?

**Comment by pcallycat** [comment]
I don't think it's home-manager, I tried to enable this on a fresh, unaltered install (outside of enabling livba, etc), and I get the same error.

**Comment by NicolasGuilloux** [comment]
As I'm not using Home-Manager, I may have miss this one. I will check right now ;)

EDIT: Okay so the issue was fixed on the `master` but no tag was made for the occasion. I'll do it right now.

As @Elyhaka gave me the ownership, I will do some changes to the repository. 
- First of all, I'm doing a big clean of the whole files to add comments, tidy things up and separate the features. 
- With that will come a new tag version naming because I find the actual one confusing and annoying to type. 
- I think I'll take some time to setup a workflow to test a system and a home-manager import to make sure everythink works before merging.

**Comment by NicolasGuilloux** [comment]
@pcallycat @savoisn Fixed in `v0.14.1`. Cheers guys!
